templates/collection/index.html

```diff
@@ -17,6 +17,13 @@
         You are previewing a collection that is not published.
       {% endblocktrans %}
     </div>
+  {% elif object.published_date is not None %}
```

Comment:
can we just `elif object.published_date`? Is there any case where we explicitly need to check if published date is None?

Reply:
I think that, there is no case where we need to check if its None, but on other hand is this condition valid? Let say we create published collection with publication date set to today + 1 day. It will say that we are previewing not published collection that will become visible on xyz which is fine, but the next day it will continue to show same information because this condition just checks if object published date is set. I'm assuming it should be additionally compared with today's date.

Reply:
Right, I think that we can remove this `elif` statement and manipulate the message in the above's `if`. `object.is_published` already checks if 1. `published_date is not None` or `published_date < date.today()`.

In such case, I think that we could construct it like:

```
if not object.is_published:
    if object.published_date:
         You are previewing a collection that will become visible on...
    else:
          You are previewing a collection that is not published...
```